app/v2/models/response/retrieveBsasAdjustments/foreignProperty/IncomeBreakdown.scala

```diff
@@ -26,7 +26,7 @@ case class IncomeBreakdown(rentIncome: Option[BigDecimal],
 
 object IncomeBreakdown {
   val fhlReads: Reads[IncomeBreakdown] = (
-    (JsPath \ "totalRentsReceived").readNullable[BigDecimal] and
+    (JsPath \ "rentReceived").readNullable[BigDecimal] and
```

Comment:
What is this change for?

Reply:
The spec says `adjustments.income.totalRentsReceived` is mapped to `adjustments.incomes.rentIncome` - this was correct before. Please change it back (or explain the reason behind this change)